views/includes/check-your-answers/beneficial-owner-individual.html

```diff
@@ -240,7 +250,7 @@
       changeLinkUrl + "/" + boi.id + OE_CONFIGS.IS_ON_SANCTIONS_LIST,
       "Individual beneficial owner " + boiFullName + " - are they on the sanctions list?",
       "change-individual-beneficial-owner-sanctions-list-button"
-    ) ]
+    ) if pageParams.isRegistration ]
```

Comment:
The text needs to be 'Are they on the UK sanctions list?' and not 'Are they on the sanctions list?'

Reply:
I agree, feel free to raise this with the BA